### ISSUE TITLE
RCS1213: Add OnRectTransformDimensionsChange to list of Unity methods

### ISSUE THREAD

**Issue by NoTuxNoBux** [dotnet/roslynator#1595]
Hi

I'm using `RCS1213` with `roslynator_suppress_unity_script_methods = true` and it works nicely, but it looks like `OnRectTransformDimensionsChange` is not part of the list of methods that are detected as Unity methods and it will still report the method as unused if it's part of a `MonoBehaviour`:

```cs
public sealed class ScaleBoxColliderToUIElement : MonoBehaviour
{
    // RCS1213 reported here:
    private void OnRectTransformDimensionsChange()
    {
    }
}
```

This method still works/exists, but it doesn't appear to be (well) documented any more, which is probably why it was missed - it can still be seen [here](https://docs.unity3d.com/2018.3/Documentation/ScriptReference/EventSystems.UIBehaviour.html).

**Comment by Kissaki** [comment]
The [RCS1213 docs](https://josefpihrt.github.io/docs/roslynator/analyzers/RCS1213/) say `roslynator_suppress_unity_script_methods` is deprecated in favor of

```
# Enable code analysis to detect Unity-specific code
roslynator_unity_code_analysis.enabled = true|false
```

Does that make a difference, or is this an issue (false positive) with that set too / still?

**Comment by NoTuxNoBux** [comment]
Thanks for the response. I indeed didn't set that yet and just tried it, but it indeed still reports `OnRectTransformDimensionsChange` as unused.
